harness/determined/exec/prep_container.py

```diff
@@ -290,6 +301,7 @@ def do_proxy(sess: api.Session, allocation_id: str) -> None:
     parser.add_argument("--rendezvous", action="store_true")
     parser.add_argument("--proxy", action="store_true")
     parser.add_argument("--notify_container_running", action="store_true")
+    parser.add_argument("--download_context_directory", action="store_true")
```

Comment:
You're following the example set by existing arguments, but in the spirit of continuous improvement, could you please give this a short [help](https://docs.python.org/3/library/argparse.html#help)?

Reply:
Looks great!